docs/static/transforming-data.asciidoc

```diff
@@ -429,27 +443,39 @@ For example:
 +
 [source,json]
 --------------------------------------------------------------------------------
-          "geoip" => {
-              "timezone" => "Europe/Moscow",
-                    "ip" => "83.149.9.216",
-              "latitude" => 55.7522,
-        "continent_code" => "EU",
-             "city_name" => "Moscow",
-         "country_code2" => "RU",
-          "country_name" => "Russia",
-              "dma_code" => nil,
-         "country_code3" => "RU",
-           "region_name" => "Moscow",
-              "location" => [
-            [0] 37.6156,
-            [1] 55.7522
-        ],
-           "postal_code" => "101194",
-             "longitude" => 37.6156,
-           "region_code" => "MOW"
-    }
+filter {
+  geoip {
+    source => "clientip"
+  }
+}
 --------------------------------------------------------------------------------
 
+<<plugins-filters-http,http filter>>::
+
+Integrates with external web services/REST APIs, and
+enables lookup enrichment against any HTTP service or endpoint.
+The <<plugins-filters-http,http filter>> is well suited to many enrichment use
+cases, such as social APIs, sentiment APIs, security feed APIs, and business
+service APIs.
++
+[source,txt]
+-----
+filter {
```

Comment:
@jsvd @karenzone we should probably verify that this config actually works as I didn't actually test it myself. We should expect some folks to simply copy, paste, and edit.